### ISSUE TITLE
Is a kOS core suposed to be able to control a vessel by its self?

### ISSUE THREAD

**Issue by TDW89** [KSP-KOS/KOS#1305]
If you launch a vessel with no probe core or command pod then you can still access the kOS terminal via the top right bar or boot script.

While running like this you can:
control steering (torque and rcs only , wheels and ailerons do not steer).
control throttle.
stage.
 - solid rockets activate.
 - liquid engines activate in staging diagram but not on vessel.
 - decouplers do not decouple.
action groups work but the things they control vary.
 - breaks, panels, etc work,
 - solid rockets work
 - most other stuff doesn't activate.

not sure which way this should work. It would be quite interesting to have vessels that could only be run by scrip (no user control just the terminal). Remote tech already fulfils this with probes out of com range though.
Or leave it as is and have it so that a core on its own is only partially functional, and force the user to set up the vessel wile it is attached to a control-able one and set up scripts specifically for 'uncontrolled' vessels.
Or of course just block all forms of control on vessels with out a control part.

**Comment by dcrockwell** [comment]
The way I see it, without a probe core there's nothing for a CPU to control. The probe core represents the "hardware controller" which kOS interfaces with. Just my 2 cents though.

**Comment by kosstarter** [comment]
I don't agree with dcrockwell. The probecore does represent the "hardware controller" but it allows the user to interface with a craft without any Kerbal involvement. A kOS core represents another type of "hardware controller", one that controls a craft with a program instead of user input. So, i.m.o a kOS core is a third type of control, next to the probe core and command pod and shouldn't need a probecore or a command pad to control a craft.

**Comment by hvacengi** [comment]
In my opinion, the probe core is the hardware that interfaces with control systems, while the kOS cup gives command to the probe core, much like your manual control input tells the probe core what to do.

I would anticipate this mechanic adjusting, and possibly getting better integration, with KSP 1.2 where they plan to expose a way for parts to require a remote connection, or pod control, in order for the part to work. We will be having discussions on how best to integrate into the new KSP feature.

**Comment by valery-labuzhsky** [comment]
I vote for kOS controlling ability. 

As far as I know the subject there is no dedicated "hardware controller" (I wouldn't bet on it though). Wires from different devices just connect to CPU (or manual input/output devices) that's it.
What's the purpose of this "hardware controller"? Is it just a place where all hardware wires are concentrated? Or does it converts communication protocols between hardware and CPU? Why not just use standard protocols everything understand? Oh, these Kermans!...
